src/index.js

```diff
@@ -0,0 +1,54 @@
+import './styles.css';
+
+const todo = [
```

Comment:
- [ ] Please make sure that you add a:- 
1- function for adding a new task (add a new element to the array) is implemented.
2- A function for editing task descriptions is implemented.
3- A function for deleting a task (remove an element from the array) is implemented.